Jenkinsfile

```diff
@@ -28,7 +28,7 @@ pipeline {
         }
         stage('Docker image') {
             steps {
-                buildDockerAndPublishImage('account-app', [rebuildImageOnPeriodicJob: false])
+                buildDockerAndPublishImage('account-app', [rebuildImageOnPeriodicJob: false, buildDockerConfig: [targetplatforms: 'linux/amd64,linux/arm64']])
```

Comment:
unrelated but it seems that `rebuildImageOnPeriodicJob` is not part of this process `buildDockerAndPublishImage` but `parallelDockerUpdatecli` instead ... or am I mistaking ?

Reply:
Good catch! I'll fix it in another PR.

Reply:
Addressed in https://github.com/jenkins-infra/account-app/pull/318/commits/fa0c9155595ec248537040c606e37e08409eeb30